flink-architecture-tests/flink-architecture-tests-test/src/test/resources/archunit.properties

```diff
@@ -0,0 +1,31 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# By default we allow removing existing violations, but fail when new violations are added.
+freeze.store.default.allowStoreUpdate=true
+
+# Enable this if a new (frozen) rule has been added in order to create the initial store and record the existing violations.
```

Comment:
Your point is fair enough. I would like to build the clear separation between production and test, so that developers do not need to jump between `flink-architecture-tests-production` and `flink-architecture-tests-production` back and forth. This file will be copied from  `flink-architecture-tests-test` and it is up to the submodule developer whether they want to remove all the commented out stuff or not. IMHO, I would not do that, because it will make the future maintenance harder if it is removed.